### PR TITLE
1207 - Allow videos(mp4, mov) to be uploaded

### PR DESCRIPTION
**Pull request by tavy88**
### Context

 Allow videos(mp4, mov) to be uploaded
 Also adjust the max file size to 50MB.

### Changes proposed in this pull request

 Allow users to upload videos(mp4, mov) as evidence. Also increase the max file size to 50MB.

Screenshot 2023-02-20 at 09 02 49

### Link to Trello card

https://trello.com/c/HbMGSRS2

### Checklist

- [x] Attach to Trello card
- [x] Rebased main
- [x] Cleaned commit history
- [x] Tested by running locally

### CODE REVIEW

File: app/validators/file_upload_validator.rb [hunk 1]
```diff
@@ -16,6 +16,8 @@ class FileUploadValidator < ActiveModel::EachValidator
     ".jpg" => "image/jpeg",
     ".jpeg" => "image/jpeg",
     ".mp3" => "audio/mpeg",
+    ".mp4" => "video/mp4",
+    ".mov" => "video/quicktime",
```
Comment (vassyz) [inline app/validators/file_upload_validator.rb:20]: Just these two? A few other formats that come to mind:  .avi , .divx , .mpg.

File: spec/validators/file_upload_validator_spec.rb [hunk 2]
```diff
@@ -35,7 +35,7 @@
   context "with a large file" do
     let(:files) { fixture_file_upload("upload1.pdf", "application/pdf") }
 
-    before { allow(files).to receive(:size).and_return(25 * 1024 * 1024) }
+    before { allow(files).to receive(:size).and_return(51 * 1024 * 1024) }
```
Comment (paulodeon) [inline spec/validators/file_upload_validator_spec.rb:38]: Maybe consider `50.megabytes + 1` here

File: app/validators/file_upload_validator.rb (hunk 1 above)
Comment (tavy88) [reply app/validators/file_upload_validator.rb:20]: Yes if you look at the ticket I think that's what we agreed with @NickGraham101

File: spec/validators/file_upload_validator_spec.rb (hunk 2 above)
Comment (tavy88) [reply spec/validators/file_upload_validator_spec.rb:38]: What would the advantage be? This test proves that whatever is over the limit makes the object invalid.

Comment (paulodeon) [reply spec/validators/file_upload_validator_spec.rb:38]: I think `50.megabytes + 1` or indeed `51.megabytes` is more readable, but I don't feel strongly about it!

**Review by gpeng — APPROVED** [review]

**Pull request merged**